### PR TITLE
Feature: Empty State Component

### PR DESCRIPTION
**Pull request by wahib-sirka**
- [x] Finished implementation of empty state for all variant

https://user-images.githubusercontent.com/104057020/179133639-662ee6f4-7b50-4a65-b158-16c0fe447375.mov

### CODE REVIEW

File: src/components/EmptyState/EmptyState.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  const { image, title, desc, button } = args;
+
+  return (
+    <EmptyStateComponent
+      title={title}
+      image={image}
+      desc={desc}
+      button={button}
+    />
+  );
```
[the remark this answers is not in the document]
Comment (wahib-sirka) [reply src/components/EmptyState/EmptyState.stories.tsx:30]: iya dinamis mar

File: src/components/EmptyState/type.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+interface BaseEmptyStateProps {
+  image?: React.ReactNode;
+  title?: string;
+  desc?: string;
```
[the remark this answers is not in the document]
Comment (wahib-sirka) [reply src/components/EmptyState/type.ts:4]: well why not, will do yah

File: src/components/EmptyState/style.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    line-height: 2.375rem;
+    text-align: center;
+    color: #262626;
+    margin-bottom: 0.5rem;
+  }
+
+  .empty-state__description {
+    margin: 0;
+    font-weight: 400;
+    font-size: 18px;
```
[the remark this answers is not in the document]
Comment (wahib-sirka) [reply src/components/EmptyState/style.ts:28]: got it

File: src/components/EmptyState/type.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (wahib-sirka) [reply src/components/EmptyState/type.ts:4]: done

File: src/components/EmptyState/style.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (wahib-sirka) [reply src/components/EmptyState/style.ts:28]: done

File: src/components/EmptyState/EmptyState.stories.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (MarioAtSirka) [reply src/components/EmptyState/EmptyState.stories.tsx]: ok cool

**Review by MarioAtSirka — APPROVED** [review]
looks good

**Pull request merged**